app/controllers/orga/mailings_controller.rb

```diff
@@ -0,0 +1,67 @@
+class Orga::MailingsController < Orga::BaseController
+  before_action :normalize_params, only: [:create, :update]
+  before_action :find_mailing, only: [:show, :edit, :update, :destroy]
+
+  def index
+    authorize! :read, :mailing
+    @mailings = mailings
+  end
+
+  def show
+    authorize! :read, :mailing
+  end
+
+  def new
+    @mailing = Mailing.new
+  end
+
+  def create
+    @mailing = Mailing.new(mailing_params)
+    if @mailing.save
+      redirect_to orga_mailing_path(@mailing), notice: 'Mailing was successfully created.'
+    else
+      render action: :new
+    end
+  end
+
+  def update
+    if @mailing.update(mailing_params)
+      redirect_to orga_mailing_path(@mailing), notice: 'Mailing was successfully updated.'
+    else
+      render action: :edit
+    end
+  end
+
+  def destroy
+    @mailing.destroy!
+    redirect_to orga_mailings_path, notice: "The mailing has been deleted"
+  end
+
+  private
+
+  def find_mailing
+    @mailing ||= Mailing.find(params[:id])
```

Comment:
Why is this `||=` rather then just `=`? Maybe I'm overseeing it, but I don't find a scenario where this would be called twice within one request response cycle...?